tests/unit/s2n_client_cert_verify_test.c

```diff
@@ -81,5 +146,51 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_cert_chain_and_key_free(chain_and_key));
     }
 
+    /*  Test: client certificate verify. */
+    {
+        struct s2n_cert_chain_and_key *chain_and_key;
+        EXPECT_SUCCESS(s2n_test_cert_chain_and_key_new(&chain_and_key,
+                S2N_DEFAULT_TEST_CERT_CHAIN, S2N_DEFAULT_TEST_PRIVATE_KEY));
+
+        struct s2n_config *config;
+        EXPECT_NOT_NULL(config = s2n_config_new());
+        EXPECT_SUCCESS(s2n_config_add_cert_chain_and_key_to_store(config, chain_and_key));
+        EXPECT_SUCCESS(s2n_config_set_async_pkey_callback(config, async_pkey_perform_op));
+        EXPECT_SUCCESS(s2n_config_set_unsafe_for_testing(config));
+        EXPECT_SUCCESS(s2n_config_set_cipher_preferences(config, "20170210"));
+
```

Comment:
It can help to put a comment on here noting what you need from the security preferences, since they're basically just a date. In this case, you need security preferences that don't support TLS1.3.

Reply:
Added a comment  👍